BUILDING_FROM_SOURCE.md

```diff
@@ -168,20 +168,20 @@ If you encounter:
 
 ```text
 error: No suitable version of LLVM was found system-wide or pointed
-       to by LLVM_SYS_160_PREFIX.
+       to by LLVM_SYS_180_PREFIX.
 ```
 
-Add `export LLVM_SYS_160_PREFIX=/usr/lib/llvm-16` to your `~/.bashrc` or equivalent file for your shell.
+Add `export LLVM_SYS_180_PREFIX=/usr/lib/llvm-18` to your `~/.bashrc` or equivalent file for your shell.
 
 ### LLVM installation on MacOS
 
-For macOS, you can install LLVM 16 using `brew install llvm@16` and then adding
-`$(brew --prefix llvm@16)/bin` to your `PATH`. You can confirm this worked by
+For macOS, you can install LLVM 18 using `brew install llvm@18` and then adding
+`$(brew --prefix llvm@18)/bin` to your `PATH`. You can confirm this worked by
```

Comment:
Confirmed this is available in home-brew and is also the current stable release
https://formulae.brew.sh/formula/llvm